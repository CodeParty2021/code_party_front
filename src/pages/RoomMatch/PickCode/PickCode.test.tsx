import React from "react";
import { shallow } from "enzyme";
import { RoomMatchPickCode } from "./PickCode";

describe("<RoomMatchPickCode />", () => {
  it("snapshot test", () => {
    const wrapper = shallow(<RoomMatchPickCode />);

    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
