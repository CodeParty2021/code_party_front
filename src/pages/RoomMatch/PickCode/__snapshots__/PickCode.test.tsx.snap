// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoomMatchPickCode /> snapshot test 1`] = `
Array [
  <div>
    <div>
      ロボットを選択する画面
    </div>
    <Link
      to="/room-match/invitation/:roomId"
    >
      マッチ画面に戻る
    </Link>
  </div>,
]
`;
